src/main/scala/gpp/ui/forms/ExternalValue.scala

```diff
@@ -0,0 +1,29 @@
+package gpp.ui.forms
+
+import japgolly.scalajs.react.Callback
+import crystal.ViewF
+import crystal.react.implicits._
+import cats.effect.Effect
+import japgolly.scalajs.react.extra.StateSnapshot
+
+trait ExternalValue[W[_]] {
```

Comment:
I guess this will let us use it from the old seqexec too

Reply:
Yes, that's the reason for doing that